### PR TITLE
Add blog post about DIGEST SHA algorithms configuration

### PR DESCRIPTION
**Pull request by skyllarr**

### CODE REVIEW

File: _posts/2023-04-04-specify-http-digest-algorithms.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: 'Configure the digest authentication mechanism to accept SHA-256 and SHA-512-256 algorithms'
+date: 20223-04-04
```
Comment (fjuma) [inline _posts/2023-04-04-specify-http-digest-algorithms.adoc]: Please update the filename and the date

File: _posts/2023-04-04-specify-http-digest-algorithms.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: 'Configure the digest authentication mechanism to accept SHA-256 and SHA-512-256 algorithms'
```
Comment (fjuma) [inline _posts/2023-04-04-specify-http-digest-algorithms.adoc]: Maybe could reword to "Configuring the Digest mechanism to make use of additional algorithms"

File: _posts/2023-04-04-specify-http-digest-algorithms.adoc (hunk 1 above)
Comment (skyllarr) [reply _posts/2023-04-04-specify-http-digest-algorithms.adoc]: fixed

File: _posts/2023-04-04-specify-http-digest-algorithms.adoc (hunk 2 above)
Comment (skyllarr) [reply _posts/2023-04-04-specify-http-digest-algorithms.adoc]: fixed. Thanks!

File: _posts/2023-05-15-specify-http-digest-algorithms.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: 'Configuring the Digest mechanism to make use of additional algorithms'
+date: 20223-05-15
```
Comment (fjuma) [inline _posts/2023-05-15-specify-http-digest-algorithms.adoc]: 2023 :)

Comment (skyllarr) [reply _posts/2023-05-15-specify-http-digest-algorithms.adoc]: Fixed!

**Pull request merged**